lib/tran_sock.h

```diff
@@ -89,7 +89,8 @@ vfu_send_error(int sock, uint16_t msg_id,
  */
 int
 vfu_recv(int sock, struct vfio_user_header *hdr, bool is_reply,
```

Comment:
Please add a vfu_recv_fds() so only those who need it have to use the more complicated version.

Reply:
ok